operators/src/source/ogr_source/mod.rs

```diff
@@ -1297,7 +1307,7 @@ where
         builder.push_time_interval(time_interval);
 
         for (column, data_type) in data_types {
-            let field = feature.field(column);
+            let field = feature.field_index(column).and_then(|i| feature.field(i));
```

Comment:
maybe we should look the index up only once (same for the other occurrences)? I know before the change to the gdal API the index was also looked up everytime but I think we weren't aware of that when we wrote this.

Reply:
[da49d4f](https://github.com/geo-engine/geoengine/pull/1048/commits/da49d4fb4bcefffba577b8bd603c1851fa96dd2c)